libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -243,6 +243,10 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             break;
         }
     }
+    if (sitl_model == nullptr) {
```

Comment:
> It's a static in the bss.
Yep... (Didn't see your comment before posting my last one :-)). Sorry about the noise